src/main/java/frc/robot/RobotContainer.java

```diff
@@ -47,17 +48,20 @@
 
   // joystick2 buttons
   private Button resetGyro;
-  private JoystickButton autoMove;
   private JoystickButton startShootin;
   private JoystickButton stopShootin;
 
   private JoystickButton intakeButton;
   private JoystickButton feedButton;
   private JoystickButton ejectButton;
 
-  // private final SlewRateLimiter m_xspeedLimiter = new SlewRateLimiter(3);
-  // private final SlewRateLimiter m_yspeedLimiter = new SlewRateLimiter(3);
-  // private final SlewRateLimiter m_rotLimiter = new SlewRateLimiter(3);
+  private final SlewRateLimiter m_xspeedLimiter = new SlewRateLimiter(3);
+  private final SlewRateLimiter m_yspeedLimiter = new SlewRateLimiter(3);
+  private final SlewRateLimiter m_rotLimiter = new SlewRateLimiter(3);
+
+  private DoubleSupplier m_translationXSupplier;
+  private DoubleSupplier m_translationYSupplier;
+  private DoubleSupplier m_rotationSupplier;
```

Comment:
need to keep these as member variables so we can pass them into commands that take driving input (lock-on to target rotation but allow driver translation)